### PR TITLE
Add whitespace/punctuation to some message descriptions

### PR DESCRIPTION
**Pull request by julthep**
Add whitespace/punctuation to some message descriptions.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description
Add whitespace/punctuation to some message descriptions.

### CODE REVIEW

File: pylint/checkers/base/comparison_checker.py [hunk 1]
```diff
@@ -76,8 +76,8 @@ class ComparisonChecker(_BasicChecker):
         "W0177": (
             "Comparison %s should be %s",
             "nan-comparison",
-            "Used when an expression is compared to NaN"
-            "values like numpy.NaN and float('nan')",
+            "Used when an expression is compared to NaN "
+            "values like numpy NaN and float('nan').",
```
Comment (DanielNoord) [inline pylint/checkers/base/comparison_checker.py:80]: This is incorrect. `numpy.Nan` is the qualified name of the class.

File: pylint/checkers/base/docstring_checker.py [hunk 2]
```diff
@@ -98,8 +98,8 @@ class DocStringChecker(_BasicChecker):
                 "type": "int",
                 "metavar": "<int>",
                 "help": (
-                    "Minimum line length for functions/classes that"
-                    " require docstrings, shorter ones are exempt."
+                    "Minimum line length for functions/classes that "
```
Comment (DanielNoord) [inline pylint/checkers/base/docstring_checker.py:101]: Like other said previously, please refrain from making such cosmetic changes.

**Review by DanielNoord — CHANGES_REQUESTED** [review]

File: pylint/checkers/base/docstring_checker.py (hunk 2 above)
Comment (julthep) [reply pylint/checkers/base/docstring_checker.py]: Got it. I just want it to be consistent with the nearby option. Already revert.

File: pylint/checkers/base/comparison_checker.py (hunk 1 above)
Comment (julthep) [reply pylint/checkers/base/comparison_checker.py]: That's my fault, now corrected.

**Review by julthep — COMMENTED** [review]
Already revise as recommended.

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**